jina/enums.py

```diff
@@ -158,6 +158,7 @@ class SocketType(BetterEnum):
     PAIR_CONNECT = 9
     ROUTER_BIND = 10
     DEALER_CONNECT = 11
+    ROUTER_CONNECT = 13
```

Comment:
13?

Reply:
connect sockets must be odd...

Reply:
MacRumor said iphone will not have 13 and go for 14